scene/gui/color_picker.h

```diff
@@ -125,6 +126,10 @@ class ColorPicker : public BoxContainer {
 	PopupMenu *shape_popup = nullptr;
 	PopupMenu *mode_popup = nullptr;
 	MenuButton *btn_shape = nullptr;
+	HBoxContainer *mode_hbc = nullptr;
+	HBoxContainer *hb_smpl = nullptr;
+	GridContainer *slider_gc = nullptr;
+	HBoxContainer *hex_hb = nullptr;
```

Comment:
I think these could get better names if they are no longer local.

Reply:
I'm happy to leave things better than how I found them - can you point me to a node that has properly named variables to model after?

Reply:
I think it's enough if you rename `hb_smpl` to `sample_hb`, to at least be consistent with naming.